linfa-kernel/Cargo.toml

```diff
@@ -25,7 +25,7 @@ features = ["std", "derive"]
 
 [dependencies]
 ndarray = "0.13"
-sprs = { version = "0.9", default-features = false }
+sprs = { version = "0.9.3", default-features = false }
```

Comment:
I changed this to be sure of being able to use the `diag` method on sparse matrices in order to make the interface uniform